### PR TITLE
fix: show focus indicator `<Checkbox>` and `<Radio>` components

### PR DESCRIPTION
**Pull request by masuP9**
## What does this change?

Changed components DOM and style for focus indicator of `<Checkbox>` and `<Radio>`. I guess that those components does not show focus indicator are unexpected results. Because those components does not have `outline: none` and `Input` components has focus style 👍

### cause

`input` elements has `opacity: 0` property, so focus indicator is transparent too.

### details

set `<Box>` styled components after input element and styled `input:focus + *`

## TODO

- [ ] make focused style of `<Checkbox>` and `<Radio>` (Temporarily styling in red).

## Storybook

- [`<Checkbox>`](https://deploy-preview-310--smarthr-ui.netlify.com/?path=/story/checkbox--all)
- [`<Radio>`](https://deploy-preview-310--smarthr-ui.netlify.com/?path=/story/radio--all)

### CODE REVIEW

File: src/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -102,7 +110,12 @@ const Input = styled.input`
   &[disabled] {
     pointer-events: none;
   }
+
+  &:focus + span {
+    box-shadow: 0 0 0 2px rgba(255, 0, 0, 0.85); /* FIXME focus style */
```
Comment (nabeliwo) [inline src/components/Checkbox/Checkbox.tsx]: Use the color here.
`rgba(0, 165, 171, 0.5)`

File: src/components/Radio/Radio.tsx [hunk 2]
```diff
@@ -109,4 +115,8 @@ const Input = styled.input`
   &[disabled] {
     pointer-events: none;
   }
+
+  &:focus + span {
+    box-shadow: 0 0 0 2px rgba(255, 0, 0, 0.85); /* FIXME focus style */
```
Comment (nabeliwo) [inline src/components/Radio/Radio.tsx]: Use the color here.
`rgba(0, 165, 171, 0.5)`

File: src/components/Checkbox/Checkbox.tsx (hunk 1 above)
Comment (masuP9) [reply src/components/Checkbox/Checkbox.tsx]: Thank you for fix color!

If `rgba(0, 165, 171, 0.5)` mean half transparent `palette.Main`, use polished opacity method like `opacity(0.5, ${palette.Main}) is better than just `rgba()`. Additionally looks good that to define outline color to Palette Property as `Outline`. What do you think about ?

https://polished.js.org/docs/#opacify

Comment (nabeliwo) [reply src/components/Checkbox/Checkbox.tsx]: That's right ✨
Since `polished` is already installed, you should use it.
Thank you for the exact proposal!!

Comment (masuP9) [reply src/components/Checkbox/Checkbox.tsx]: @nabeliwo I add commit for translate to half transparent color from `Main` by transparentize (I made mistake `opacify` as make transparent method.) Please confirm again 🙏

**Pull request merged**